### ISSUE TITLE
The loss

### ISSUE THREAD

**Issue by stvsd1314** [sfujim/TD3_BC#5]
Hello, want to know if this loss function "-lmbda * Q.mean() + F.mse_loss(pi, action)" setup is reasonable, because when Q is greater than 0, the TD3 loss term becomes a constant, which essentially degenerates into Behavior Cloning (BC), right?
